### DISCUSSION TITLE
EJBCA Community Migration Issue: Missing "subject_dn_OI" Field

### DISCUSSION THREAD

**Discussion by ramiabdallah1512** [Keyfactor/ejbca-ce#603] [Q&A]
Hello,

Following the migration of EJBCA Community from version 7.4.3.2 to version 7.11 Community, the "subject_dn_OI" (Other Name - Organization Identifier) field no longer appears when requesting new certificates. Could you please help me to solve this issue?

Thank you in advance for your help.

**Comment by primetomas** [comment]
Sounds like something you have added yourself to 7.4.3.2? What kind of organization identifier is this?

**Comment by ramiabdallah1512** [comment]
hELLO? 

i found that we have manually added this configuration on ejbca [7.4.3.2](https://7.4.3.2/), i tried to copy it on my new version 7.11 but it didn't works, i did not find the OI in ejbca interface...

----------------------------------------------------

Mejdi Lassoued 05/06/2024 18:37 • ## Add OI : Organisation Identifier:
cd /opt/ejbca/src/java/
vim [profilemappings.properties](https://profilemappings.properties/)
# Add line:

DN;ORGANIZATIONIDENTIFIER;106;OI;106;ORGANIZATIONIDENTIFIER;DN_PKIX_OI
----

vim /opt/ejbca/modules/admin-gui/resources/languages/languagefile.en.properties
# Add line:
----

DN_ETSI_ORGANIZATION_IDENTIFIER = organizationIdentifier, Organization identifier

DN_PKIX_OI = organizationIdentifier, Organization identifier

----

**Reply by primetomas** [reply]
This Organization Identifier is specific for eIDAS TSPs (ETSI EN 319-412-1organization legal identifiers) was added as an EJBCA Enterprise feature since EJBCA 6.5 (some 10 years ago). Enterprise support is available for this feature.

**Comment by ramiabdallah1512** [comment]
Hello Tomas,
how can we update our 7.11 from community to entreprise, we need this Organization Identifier for our certifs.
and can you confirm that this upgrade to entreprise will resolve our problem without any impact on our production environment ?
also i want to know if with certextensions we can find a solution to this eidas recommandation,
thank you

**Reply by primetomas** [reply]
Feel free to message me directly and I can find the correct contact points. If you have LinkedIn: https://www.linkedin.com/in/tgustavsson/

Certificate extensions can not be used for Subject DN fields. It's something completely different.

Cheers,
Tomas

**Comment by ramiabdallah1512** [comment]
hello Tomas,

what are the prerequisite to update our ejbca 7.11 to entreprise ? is it a rapid change without impact ? with this entrprise release we will have the Organization Identifier parameter ?
could you support us to make this change ?

**Reply by primetomas** [reply]
Absolutely we will support you.
